docs/develop/topics/administration_panel.md

```diff
@@ -0,0 +1,432 @@
+# Administration panel views
+
+## Summary
+
+The following document is a developer guide to the invenio administration panel, detailing its programmatic interface and usage.
+
+## Intended audience
+
+This guide is intended for developers of InvenioRDM.
+
+## Overview
+
+Invenio administration panel gives the administrator a set of tools to effectively manage an instance of InvenioRDM. 
+
+## Domain dictionary
+
+The following terms are introduced to facilitate defining the domain:
+
+- `administration panel` - the interface panel enabling a `manager` to administrate the instance of InvenioRDM in a developer-independent way.
+- `administrator` - a person with domain knowledge, with a special set of permissions, able to manage an InvenioRDM instance, not necessarily having developer skills.
+- `frontsite`, `end user interface` - currently known InvenioRDM interface, accessible by anonymous and logged-in users, without administrator role.
+- `administration view` - subpages of the administration panel
+
+## Administration panel
+
+![Administration Panel](./img/administration/administration_wireframe.png)
+
+An instance developer can register new entries in the menu (see backend [RFC](https://github.com/inveniosoftware/rfcs/pull/67)). A menu entry will display the associated views in the panel section. The navbar menu layouts are customizable to meet the needs of the InvenioRDM instance.
+
+### Architecture
+
+![Architecture diagram](./img/administration/administration_architecture.png)
+
+Administration views can be used "out of the box", meaning that some functionalities are provided as long as the installed views follow the architecture mentioned below. E.g. to implement a list page for a resource, its view must be of type `AdminResourceBaseView`.
+
+More information on views, e.g. how they can be customized and extended, is provided [below](#list-view).
+
+### Views
+
+#### Create a resource based administration view
```

Comment:
`Create a resource based view`

Reply:
Are the menu/submenus correct?
![Screenshot 2022-10-05 at 22 23 28](https://user-images.githubusercontent.com/2089455/194156255-6d50bfce-16d1-4788-a855-e99b63452a9f.jpg)